### PR TITLE
add cowsay

### PR DESCRIPTION
**Pull request by bwolfson97**
This adds a cow saying moo

### CODE REVIEW

**Comment by bwolfson97** [comment]
Closes #32

File: cowsay.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+ _____
+< moo >
+ -----
+        \   ^__^
+         \  (oo)\_______
+            (__)\       )\/\
+                ||----w |
```
Comment (just-ma) [inline cowsay.txt:7]: udderly great code

**Review by just-ma — APPROVED** [review]
LGTM

File: cowsay.txt (hunk 1 above)
Comment (bwolfson97) [reply cowsay.txt:7]: Thank you

**Pull request merged**